GPX/GPXRoot.m

```diff
@@ -93,6 +96,24 @@ + (GPXRoot *)rootWithCreator:(NSString *)creator;
     return root;
 }
 
++ (NSArray<NSString *> *)keywordsArrayFromString:(NSString *)keywordString {
+    if (!keywordString) {
+        // return nil only for nil strings, to differentiate between empty and nil keywords
+        return nil;
+    }
+
+    NSArray *keywords = [keywordString componentsSeparatedByString:@","];
```

Comment:
If there's only one string (without comma), the array contains that one string, correct?